vimrc

```diff
@@ -46,12 +50,14 @@ augroup vimrcEx
   autocmd BufRead,BufNewFile .{jscs,jshint,eslint}rc set filetype=json
 
   " ALE linting events
-  set updatetime=1000
-  let g:ale_lint_on_text_changed = 0
-  autocmd CursorHold * call ale#Lint()
-  autocmd CursorHoldI * call ale#Lint()
-  autocmd InsertEnter * call ale#Lint()
-  autocmd InsertLeave * call ale#Lint()
+  if v:version >= 800
```

Comment:
I'm not really thrilled with duplicating all of that. Blargh.

Based on my reading of the script, Ale should already print a useful message. Is that the experience of folk who tried it on vim < 8?

Should we only guard the auto commands and then let the plugin do our warning for us? Or is it more clear if we explicitly say "the thoughtbot dotfiles require..."